packages/sdk/src/Sdk.ts

```diff
@@ -69,16 +68,44 @@ export class Sdk {
    * @param currency - instance of {@link Currency}
    * @param account - user address, wrapped in {@link Address}
    */
-  async getAllowanceFor(
-    currency: Currency,
-    account: Address
-  ): Promise<BigNumber> {
+  getAllowanceFor(currency: Currency, account: Address): Promise<BigNumber> {
     const router: Address = CONNEXT_ADDRESS[currency.chainId];
     return currency
       .setConnection(this._configParams)
       .allowance(account, router);
   }
 
+  /**
+   * Retruns the token balances of an address in a batch.
+   * Throws an error if `chainId` is different from each `token.chainId`.
+   *
+   * @param tokens - array of {@link Token} from the same chain
+   * @param account - user address, wrapped in {@link Address}
+   * @param chainId - ID of the chain
+   */
+  getBatchTokenBalances(
```

Comment:
```suggestion
  getBalancesFor(
```

Suggestion to be consistent with `getBalanceFor`

Reply:
You're right about pointing out the consistency and tbh my first version of this method was like yours. However, there's a slight difference between this one and the others: this accepts `Token`s and the other `Currency`. If you think this difference doesn't justify a change in the naming suite, I'll be happy to go with `getBalancesFor`.

Reply:
In that case maybe getTokensBalanceFor() ? My main concern was about the `batch` in the name, because it's redundant with the typing

Reply:
let's go then with `getTokenBalancesFor()`